apps/force/src/App.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <Molgenis id="__top" v-model="session">
+    <router-view :session="session" :page="page"></router-view>
+  </Molgenis>
+</template>
+
+<script setup lang="ts">
```

Comment:
I think you also need to add a tsconfig in order for ts to get the highlighting correct in your editor. You can copy one from one of the other apps e.g. molgenis-components.